lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -4458,7 +4471,37 @@ sub process_index_phrases {
     T_BEGIN, @expansion, T_END);
   return @expansion; }
 
-DefMacro('\index{}', \&process_index_phrases);
+# read verbatim, as perl latex.ltx \@sanitize;
+# useful for \index (maybe others?)
+DefParameterType('SanitizedVerbatim', sub {
+    my ($gullet) = @_;
+    $gullet->readUntil(T_BEGIN);
+    # crucial: deactivate the backslash to avoid activating command sequences
+    # chars switched to CC_OTHER by \@sanitize: ' ', '\\', '$', '&', '#', '^', '_', '%', '~'
+    # some of those are already in state's "SPECIALS", so only adding the rest:
+    StartSemiverbatim(' ', '\\', '%');
+    my $arg = $gullet->readBalanced(1);
+    EndSemiverbatim();
+    # now that we have the semiverbatim tokens, retokenize with the standard catcode scheme,
+    # this may seem like wasted work, but it avoids very unfortunate error propagation in cases
+    # where the \index argument was malformed for one reason or another.
+    #
+    # the strangeness comes from the original TeX workflow requiring multiple conversion calls,
+    # alongside a call to the `makeidx` binary, which we don't do in latexml. This parameter type
+    # emulates one important aspect implied by those steps.
+    $arg = Tokenize(ToString($arg));
+    return $arg; },
+  beforeDigest => sub {
+    $_[0]->bgroup;
+    MergeFont(family => 'typewriter'); },
```

Comment:
this will only have effect if the parameter is used in a Constructor; are you anticipating this?

Reply:
I was not, thank you for raising awareness. But it also was cargo-culted without good reason (from `HyperVerbatim`), so I could erase it safely.